CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* None.
+* Swift: Added `List.invalidated`, which returns if the given `List` is no longer
```

Comment:
Minor point: I like denoting instance methods/properties with `List().invalidated` and class/static methods/properties as `List.invalidated`, but I may be in the minority here.